lib/perl/Genome/Model.pm

```diff
@@ -471,6 +471,8 @@ sub __display_name__ {
 sub create {
     my $class = shift;
 
+    no warnings 'redefine';
+    local *__errors__ = sub { () };
```

Comment:
Overriding ```UR::Context::Transaction->changes_can_be_saved``` would be a gigantic break of encapsulation.  Adding an option to check validity isn't too onerous; I'll go with that.